<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 247, 237);
	gridline-color: rgb(170, 85, 255);
	border-radius: 25Px;
	border: 2px solid rgb(170, 85, 255);
}
QPushButton:hover{
	
	background-color: rgb(255, 187, 255);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(170, 170, 255);
}</string>
    </property>
    <property name="text">
     <string>당신의 투자 유형을 확인하고 싶으면 클릭</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_investment_type">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>181</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
